float64 command_vx
float64 command_vy
float64 command_vw
float64[] joint_speed
float64[] joint_speed_filtered
float64[] joint_speed_error
float64[] joint_commanded_effort
float64[] joint_applied_effort
int32[] joint_stall
string[] joint_name

